core/modules/qproc/QuerySession.cc

```diff
@@ -298,13 +299,21 @@ void QuerySession::finalize() {
 
 
 QuerySession::QuerySession(Test& t)
-    : _css(t.css), _defaultDb(t.defaultDb), _sqlConfig(t.sqlConfig) {
+    : _css(t.css), _defaultDb(t.defaultDb) {
+    sql::SqlConfig sqlConfig(t.sqlConfig);
+    _databaseModels = DatabaseModels::create(sqlConfig, sqlConfig);
     _initContext();
 }
 
 
+QuerySession::QuerySession() {
+    sql::SqlConfig sqlConfig(sql::SqlConfig::MOCK);
+    _databaseModels = DatabaseModels::create(sqlConfig, sqlConfig);
```

Comment:
passing in the config twice is an odd pattern. Maybe an overload taking only one config instance is appropriate?

Reply:
Same config for both local and master, since local is also master. I'm not sure adding another create statement helps but I'll add a comment.